src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,49 @@
+package coffee.order;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CoffeeOrderBoard {
+    private final List<Order> orders;
+
+    public CoffeeOrderBoard() {
+        orders = new ArrayList<>();
+    }
+
+    public void add(Order order) {
+        int orderNumber = orders.size() + 1;
+        order.setOrderNumber(orderNumber);
+        orders.add(order);
+    }
```

Comment:
incorrect implementation order number is always unique,  in your case different person can get a same order number

Reply:
fix the one